test/platforms/aws-eks/README.md

```diff
@@ -0,0 +1,87 @@
+
+# Context
+
+Terraform templates to quickly create an EKS cluster with a managed node group. This is not a reference setup! It's a vanilla setup to be used when attempting to replicate issues and/or to test new features.
+
+⚠️ Do not use this setup in production.
+
+## Pre-requisites
+
+- Terraform v1.3+ installed locally.
+- an AWS account
+- the AWS CLI v2.7.0/v1.24.0 or newer, installed and configured
+- AWS IAM Authenticator
+- kubectl v1.24.0 or newer
+
+<details>
+    <summary>Download & Authenticate</summary>
+
+```bash
+brew install awscli aws-iam-authenticator terraform
+```
+
+```bash
+# Configure & authenticate AWS CLI
+# This will vary based on your AWS account and IAM setup
+```
```

Comment:
Does this need to be a code block? Do we a link that can help as a starting point if someone is trying to figure this one out?